toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -806,6 +813,57 @@ func validateOutput(baseConfigPath string, output imagecustomizerapi.Output, out
 	return nil
 }
 
+func validateStorage(storage *imagecustomizerapi.Storage) error {
+	if storage == nil {
+		return nil
+	}
+
+	err := validateVeritySignatureInjection(storage.Verity, storage.FileSystems)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func validateVeritySignatureInjection(verityList []imagecustomizerapi.Verity,
```

Comment:
This should be handled in `Storage.IsValid`.

Actually, more specifically it should be handled `ValidateVerityMounts`. `ValidateVerityMounts` is called by both `Storage.IsValid` and `validateVerityMountPaths`, `validateVerityMountPaths` is used when verity is initialized on pre-created partitions.